2-mandatory/4-people-i-know.js

```diff
@@ -402,7 +414,16 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+let coll = [];
+for (let friend in friends) {
+  coll.push(...friends[friend].colleagues);
+}
 
+for (let good of coll) {
```

Comment:
It would be grea tif we didn;t need to loop twice. Why not put the if statement in the first loop.. and use forEach()